Testscripts/Linux/docker_dotnet_app.sh

```diff
@@ -48,7 +49,10 @@ InstallDotnetSDK() {
                 apt-get update
                 apt-get install apt-transport-https
                 apt-get update
-                apt-get install -y dotnet-sdk-3.1
+                if [[ $os_RELEASE = '14.04' ]]; then
+                    SDK_VERSION="2.2"
```

Comment:
Add comments here, why 1404 supports only 2.2

Reply:
the max version I found from 1404 repo is 2.2

Reply:
yes, add it in code comments, and specify where the information is get from.